### DISCUSSION TITLE
Open Food for Gastro

### DISCUSSION THREAD

**Discussion by minhkhoinguyen** [Blair2004/NexoPOS#290] [Ideas]
Description: Open Food is used when customers request to buy products that are not available at the restaurant or not managed in the warehouse. For example, the restaurant doesn’t sell tobacco or alcohol but customers are requested to use them. We will buy the products for the customers (debit cash), then charged the bill. Open Food is different from the Service charge. The service charge will be charged when there is a processing operation (related to the management of ingredients: sugar, salt, other spices) or paid to serve.

End-user:
- Cashier can quickly manipulate on POS. The product option is saved to the database as a normal dish or only for the current order.
- Open Food will appear in sales reports, accounting reports. This helps accountants accurately control sales income.
![WhatsApp Image 2021-01-20 at 00 26 13](https://user-images.githubusercontent.com/12565814/131878568-42bd5f0d-f1ef-43db-b0d7-9c740a1d318f.jpeg)

It in same as quick product you are doing

**Comment by Blair2004** [comment]
Tihs is not applicable to NexoPOS 4.x. And you can use the "+ item" button for that. This is out of topic thread.

**Reply by Blair2004** [reply]
We've recently added a similar feature to NexoPOS 4.x that will be available on version 4.5.

**Reply by minhkhoinguyen** [reply]
Open Food is not save to food list in databse. It only use for bill of recent order
